dspace/config/registries/relationship-formats.xml

```diff
@@ -12,127 +12,279 @@
     <dc-type>
         <schema>relation</schema>
         <element>isAuthorOfPublication</element>
-        <scope_note></scope_note>
+        <scope_note>Contains all AUTHOR uuids that are related to the current PUBLICATION, from the point of view of current PUBLICATION. Internally used by DSpace. Do not manually add, remove or edit values.</scope_note>
```

Comment:
While I understand what you are trying to say in this `scope_note` (and others similarly worded), I find this description very confusing.  I think the "point of view" phrase is just confusing things.

I wonder if we should reword this to be something like:
> Contains all AUTHOR uuids that the current PUBLICATION links to via a relationship. In other words, this stores all relationships pointing from the current PUBLICATION to an AUTHOR. Internally used by DSpace. Do not manually add, remove or edit values.

Then, for the corresponding `latestForDiscovery` field, we could reword its scope note to say something like:
> Contains all AUTHOR uuids of AUTHORs which link to the current PUBLICATION via a "latest" relationship. In other words, this stores all relationships pointing to the current PUBLICATION from an AUTHOR. Internally used by DSpace to support versioning. Do not manually add, remove or edit values.

OR can `latestForDiscovery` be more easily described as the "inverse relationship"?  For example
> Contains all AUTHOR uuids of AUTHORs which link to this PUBLICATION via inverse relationship (isPublicationOfAuthor). Internally used by DSpace to support versioning....

Are those descriptions accurate?  I just feel the terminology "from the point of view of" is very confusing...as I don't understand what is meant by "point of view".  I'm worried it just won't be understandable to others looking at these scope notes.

Reply:
This has been merged now